<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xray_imaging_parameters</class>
 <widget class="QWidget" name="xray_imaging_parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Xray imaging parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="ff_gbox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="ff_gridlayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item row="6" column="3">
         <widget class="QListWidget" name="detector_distance_listwidget">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="ff_offset_a_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="ff_offset_a_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Offset X (mm):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="ff_offset_b_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Offset Y (mm):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ff_pre_cbox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Take pre flat field</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="ff_offset_c_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignTop">
         <widget class="QLabel" name="detector_distance_label">
          <property name="text">
           <string>Detector distance (mm):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ff_num_images_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="ff_apply_cbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Apply FF to live view</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="store_data_cbox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Check to use energies from energy scan</string>
          </property>
          <property name="text">
           <string>Store data</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ff_post_cbox">
          <property name="text">
           <string>Take post flat field</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="ff_num_images_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Number of FF images:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="ff_offset_b_ledit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="ff_offset_c_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Offset Z (mm):</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="add_button">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="remove_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="6" column="1" alignment="Qt::AlignTop">
         <widget class="QLineEdit" name="detector_distance_ledit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="ff_ssim_cbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>FF ssim</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
